docs/VictoriaLogs/vmalert.md

```diff
@@ -0,0 +1,160 @@
+---
+weight: 10
+title: vmalert
+menu:
+  docs:
+    parent: "victorialogs"
+    weight: 10
+aliases:
+- /VictoriaLogs/vmalert.html
+---
+VictoriaLogs provides log stats APIs [`/select/logsql/stats_query`](https://docs.victoriametrics.com/victorialogs/querying/#querying-log-stats)  and [`/select/logsql/stats_query_range`](https://docs.victoriametrics.com/victorialogs/querying/#querying-log-range-stats), which return the log stats in the format compatible with [Prometheus querying API](https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries). This allows users using VictoriaLogs as the datasource of [vmalert](https://docs.victoriametrics.com/vmalert/) and creating alerting and recording rules using [LogsQL](https://docs.victoriametrics.com/victorialogs/logsql/).
+
+Note: This page only provides integration instructions for vmalert with VictoriaLogs, see full textbook of vmalert [here](https://docs.victoriametrics.com/vmalert).
+
+## Configuration
+
+Run vmalert with the `-rule.defaultRuleType=vlogs` flag.
+```
+./bin/vmalert -rule=alert.rules \            # Path to the file with rules configuration.
+    -datasource.url=http://localhost:9428 \  # VictoriaLogs address.
+    -rule.defaultRuleType=vlogs \                 # Change default rule type to VictoriaLogs.
+    -notifier.url=http://localhost:9093 \    # AlertManager URL (required if alerting rules are used)
+    -remoteWrite.url=http://localhost:8428 \ # Remote write compatible storage to persist rules and alerts state info (required if recording rules are used)
+    -remoteRead.url=http://localhost:8428 \  # Prometheus HTTP API compatible datasource to restore alerts state from
+
+```
+
+![vmalert](vmalert_victorialogs.webp)
+{width="500"}
+
+For a complete list of command-line flags, visit https://docs.victoriametrics.com/vmalert/#configuration. The following flags are specifically related to the VictoriaLogs datasource:
+
+```
+-datasource.applyIntervalAsTimeFilter
+  Only works for VictoriaLogs rules. Whether to apply the evaluation interval as a time filter for the rules. (default false)
+-rule.defaultRuleType
+  Default type for rule expressions, can be overridden by type parameter inside the rule group. Supported values: "graphite", "prometheus" and "vlogs". 
+  Default is "prometheus", change it to "vlogs" if all of the rules are written with LogsQL.
+-rule.evalDelay time
+   Adjustment of the time parameter for rule evaluation requests to compensate intentional data delay from the datasource. Normally, should be equal to `-search.latencyOffset` (cm d-line flag configured for VictoriaMetrics single-node or vmselect).
+   Since there is no intentional search delay in VictoriaLogs, `-rule.evalDelay` can be reduced to a few seconds to accommodate network and ingestion time.
+```
+
+### rule file
+
+Check how to write the rule file [here](https://docs.victoriametrics.com/vmalert/#groups).
```

Comment:
This section is confusing. I'd rename it to "Quick Start" and will provide a basic configuration example.
I'd recommend adding something similar to the following description:
```
Configure vmalert in the same way as suggested [here](https://docs.victoriametrics.com/vmalert/#quickstart).
As ` -datasource.url` specify URL of VictoriaLogs. Rules in files specified via `-rule` flag should contain groups of `vlogs` type and expressions in LogsQL format. For remotewrite and remoteread destinations we recommend using VictoriaMetrics.
```

Reply:
And then provide a copy&paste example of configuration in the same way as QuickStart suggests.

Reply:
Please check the latest doc and advise.